lxd/cluster/config.go

```diff
@@ -218,6 +223,7 @@ func configGet(cluster *db.Cluster) (*Config, error) {
 var ConfigSchema = config.Schema{
 	"backups.compression_algorithm":  {Default: "gzip", Validator: validateCompression},
 	"cluster.offline_threshold":      {Type: config.Int64, Default: offlineThresholdDefault(), Validator: offlineThresholdValidator},
+	"cluster.image_sync_node_count":  {Type: config.Int64, Default: imageSyncNodeCountDefault(), Validator: imageSyncNodeCountValidator},
```

Comment:
How about `cluster.image_minimal_replica`? We don't like `node` being mentioned anywhere outside of internal code.